src/auth/authRouter.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+import User from './user';
+import express from 'express';
+import auth from './middleware';
+import oauth from './lib/oauth';
+
+const authRouter = express.Router();
+
+authRouter.post('/register', (req, res, next) => {
+  let user = new User(req.body);
+  user.save()
+    .then(user => {
+      res.send(user.generateToken());
+    })
+    .catch(next);
+});
+
+authRouter.get('/signin', auth, (req, res, next) => {
+  res.cookie('token', req.token);

```

Comment:
As we know now, in the future you will want to name your cookies something like 'X-Rugbyscout-token'. The general template is 'X-<app-name>-<cookie-name>'.